### DISCUSSION TITLE
My block was given to someone else

### DISCUSSION THREAD

**Discussion by Adam-Student** [Chia-Network/chia-blockchain#8180] [Support]
Hello, I found the proof of the block in solo mining, there are photos, but due to the fact that my processor was busy plotting new NFT rafts, the response time increased to 6 seconds, and my 2 coins did not arrive to me. How can I get them?

**Comment by emlowe** [comment]
You didn't farm the block why do you want the rewards?

**Reply by Adam-Student** [reply]
I farm the block, 730 OG rafts farm the block solo

**Reply by emlowe** [reply]
If you think you farmed the block - look at a block explorer - see the plot public key information for the plot that farmed the block and check if you have that plot. Also check the farmer reward address for the block to see if it matches your wallet address

Also your logs will tell you whether you farmed the block (well, at least if you farmed the unfinished block and sent it to the timelord) - find a proof is not equivalent to farming a block

**Comment by BrandtH22** [comment]
@Adam-Student , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
